lib/helpers.rb

```diff
@@ -55,7 +55,7 @@ def vote_for(obj)
   def flag_as_abuse(obj)
     raise ArgumentError, t(:user_id_is_required) unless user
     # If this is the first abuse flag, count it, otherwise don't increment the count
-    obj.author.update_stats_for_course(obj.course_id, active_flags: 1) unless obj.abuse_flaggers
+    obj.author.update_stats_for_course(obj.course_id, active_flags: 1) unless obj.abuse_flaggers.empty?
```

Comment:
This should be `if obj.abuse_flaggers.empty?` instead of `unless`, if I understand correctly.